applications/ParticleMechanicsApplication/particle_mechanics_application.cpp

```diff
@@ -183,9 +183,9 @@ namespace Kratos
 
         // Registering Constitutive Laws
         // CL: Linear Elastic laws
-        KRATOS_REGISTER_CONSTITUTIVE_LAW("LinearElastic3DLaw", mHyperElastic3DLaw);
-        KRATOS_REGISTER_CONSTITUTIVE_LAW("LinearElasticPlaneStress2DLaw", mHyperElasticPlaneStrain2DLaw);
-        KRATOS_REGISTER_CONSTITUTIVE_LAW("LinearElasticPlaneStrain2DLaw", mHyperElasticPlaneStrainUP2DLaw);
+        KRATOS_REGISTER_CONSTITUTIVE_LAW("LinearElastic3DLaw", mLinearElastic3DLaw);
```

Comment:
Which are the problems with the ones already in the StructuralMechanicsApplication